fwb/app/(auth)/(routes)/sign-up/[[...sign-up]]/page.tsx

```diff
@@ -1,5 +1,2112 @@
-import { SignUp } from "@clerk/nextjs";
- 
+"use client";
```

Comment:
When I go through the sign-up and make a new account after I verify it redirects me back to clerks own sign-in component. The account I created also doesn't get registered into Clerk.